maestro-cli/src/main/java/maestro/cli/view/TestSuiteStatusView.kt

```diff
@@ -63,11 +69,11 @@ object TestSuiteStatusView {
                 )
 
                 if (canceledFlows.isNotEmpty()) {
-                    println("(${canceledFlows.size} ${flowWord(canceledFlows.size)} Canceled)")
+                    PrintUtils.warn("${canceledFlows.size} ${flowWord(canceledFlows.size)} Cancelled (Internal Maestro Cloud Error)")
```

Comment:
Let's be consistent with the spelling of `canceled` in terms of how many `l` we use. In the console we use one l and say `canceled` so let's use the same here.

<img width="399" alt="image" src="https://github.com/mobile-dev-inc/maestro/assets/6113675/0af8ce07-b2a6-4ab7-b79b-d37741be1e02">